include/translatable-object.php

```diff
@@ -373,12 +373,7 @@ public function get_objects_with_no_lang( $limit, array $args = array() ) {
 			return array();
 		}
 
-		$sql = $this->get_objects_with_no_lang_sql( $language_ids, $limit, $args );
-
-		if ( empty( $sql ) ) {
-			return array();
-		}
```

Comment:
Why remove this?

Reply:
This is exactly what motivated this PR as I was surprised by the fact that `$sql` could be empty. So the purpose of this PR is in fact to make sure that `get_objects_with_no_lang_sql()` always returns a non empty SQL query.